openapi-v1.yaml

```diff
@@ -1108,6 +1108,16 @@ definitions:
         $ref: "#/definitions/ResultFormat"
         description: type of results (native, i.e cloud pickle or json or arrow)
         x-omitempty: true
+      task_graph_uuid:
+        type: string
+        x-omitempty: true
+        description: >
+          If set, the ID of the log for the task graph that this was part of.
```

Comment:
You can omit `If set`

Reply:
I think it makes sense to keep this here, because most `ArrayTask`s are not part of a task graph. Does that make sense to you, or do you still find it unnecessary?